src/test/isolation2/output/resgroup/resgroup_startup_memory.source

```diff
@@ -0,0 +1,38 @@
+-- start_ignore
+! gpconfig -c runaway_detector_activation_percent -v 20;
+! gpstop -rai;
+
+CREATE OR REPLACE FUNCTION resGroupPalloc(float) RETURNS int AS '/home/kds/gpdb/src/test/isolation2/../regress/regress.so', 'resGroupPalloc' LANGUAGE C READS SQL DATA;
```

Comment:
Path remained the same (without generic variable), but the tests have passed?

Reply:
Yes, I did it intentionally. Compilation path is defined during configuration stage, so it differs on different machines. Having such statement outside `ignore` block will cause this test to fail if you compilation path is not the same compared to the default path on ci. If this statement fails to complete, it's not a problem either, because those statements which use this function will indicate the problem

Reply:
What's about using `@abs_builddir@` and `@DLSUFFIX@` used in input, in output as well? It's replaced in runtime by `pg_regress`. The paths don't have to be inside ignore block that way.

Reply:
Done